clients/src/main/resources/common/message/RequestHeader.json

```diff
@@ -37,6 +37,9 @@
     // Since the client is sending the ApiVersionsRequest in order to discover what
     // versions are supported, the client does not know the best version to use.
     { "name": "ClientId", "type": "string", "versions": "1+", "nullableVersions": "1+", "ignorable": true,
-      "flexibleVersions": "none", "about": "The client ID string." }
+      "flexibleVersions": "none", "about": "The client ID string." },
+    // Could not set default to null, because not all versions of this field are nullable.
```

Comment:
Why not just make all versions of the field nullable by setting a `nullVersions: "2+"`?

Reply:
Sg!